### ISSUE TITLE
The console reporter does not handle nested tests appropriately

### ISSUE THREAD

**Issue by fezzik1620** [onury/grunt-jasmine-nodejs#10]
When you have nested tests (one describe within another) they do not show up under the right parent, "describe".  When the last test for a given describe completes, everything before it shows up under that describe statement.

Here's what I mean.  For the same test, here is the console output:
![image](https://cloud.githubusercontent.com/assets/11236701/6642294/9b2e250e-c971-11e4-8044-4da40ea07bab.png)

And this is the jUnit report for the same test.
![image](https://cloud.githubusercontent.com/assets/11236701/6642321/c819ac64-c971-11e4-8bb3-22ac5443d8ea.png)

Looking at the jUnit results you can see that there are only two tests under the "MyStats page" which is actually nested inside the outer "SRM page".  Here's a snippet showing the structure of my test:
```
describe("The SRM page", function () {
    it("should load.", function (done) {
        ...
    });

    it("should redirect back to the page and load QView.", function (done) {
        ...
    });

    it("should load the MyStats page.", function (done) {
        ...
    });
    
    describe("The MyStats page", function () {
        it("should have an SR Closes button.", function (done) {
            ...
        });

        it("should load for the logged in user.", function (done) {
            ...
        });
    }); //end of the MyStats page suite

    it("closed.", function (done) {
        ...
    });
});
```

**Note:** Issue #7 would satisfy me as the jasmine-reporters terminal reporter does handle nested tests.

**Comment by onury** [comment]
Hi, I'll look into this asap and let you know.
Thanks for the detailed report!

**Comment by onury** [comment]
This is fixed in the latest version 1.3.0
A completely new console reporter. Please let me know if this works for you.
Also enabled terminal reporter for people used to it.
